docs/managed-namespace.md

```diff
@@ -2,7 +2,7 @@
 
 You can install `argo-events` in either cluster scoped or namespace scoped configuration, accordingly you need to set up ClusterRole or normal Role for service account `argo-events-sa`.
 
-In namespace scope installation, you must run `eventbus-controller`, `eventsource-controller` and `sensor-controller` with `--namespaced`. If you would like to have the controllers watching a separate namespace, add `--managed-namespace` as well.
+In namespace scope installation, you must run `controller-manager` deployment with `--namespaced`. If you would like to have the controller watching a separate namespace, add `--managed-namespace` as well.
```

Comment:
@whynowy maybe we should add some note for older versions (< v1.7) ?

Reply:
Good point!